packages/storybook-builder-vite/package.json

```diff
@@ -17,9 +17,11 @@
         "@storybook/csf-tools": "^6.3.3",
         "@storybook/source-loader": "^6.3.12",
         "@vitejs/plugin-react": "^1.0.8",
+        "ast-types": "^0.14.2",
         "es-module-lexer": "^0.9.3",
         "glob": "^7.2.0",
         "glob-promise": "^4.2.0",
+        "react-docgen": "6.0.0-alpha.0",
```

Comment:
I'm using 6.0.0-alpha.0 so it supports resolving of imports. My hope is that this means we don't have to implement react-docgen-typescript although I have a working version of that too: https://github.com/joshwooding/storybook-builder-vite/tree/explore-react-docgen-typescript